dnsdist/init.sls

```diff
@@ -3,12 +3,18 @@
 #
 {% if 'dnsdist' in salt['pillar.get']('netbox:tag_list', []) %}
 
+dnsdist-repo-key:
+  cmd.run:
+    - name: "curl https://repo.powerdns.com/FD380FBB-pub.asc | gpg --dearmor > /usr/share/keyrings/FD380FBB-keyring.gpg"
+    - creates: /usr/share/keyrings/FD380FBB-keyring.gpg
```

Comment:
I'd give this one a more descriptive name on the disk so you can see from the filename which repository it belongs to, like the other keys. So e.g. `powerdns-keyring.gpg`
Same for the recursor at the bottom

Reply:
All right. Done so